README.md

```diff
@@ -1,11 +1,15 @@
-# Spacemesh CLI Wallet
+# Spacemesh CLIWallet
 
 ## Overview
-A basic reference Spacemesh CLI Wallet designed to work together with a [go-spacemesh full node](https://github.com/spacemeshos/go-spacemesh).
+A basic reference console Spacemesh Wallet designed to work together with a [go-spacemesh full node](https://github.com/spacemeshos/go-spacemesh).
 
-The CLI Wallet is designed for developers who want to hack on the Spacemesh platform. For non-devs we recommend using Smapp - the Spacemesh App. [Smapp](https://github.com/spacemeshos/smapp) is available for all major desktop platforms and includes a wallet.
+The CLIWallet is designed for developers who want to hack on the Spacemesh platform. For non-devs we recommend using Smapp - the Spacemesh App. [Smapp](https://github.com/spacemeshos/smapp) is available for all major desktop platforms and includes a wallet.
 
 ## Functionality
+The wallet is a Spacemesh node api 2.0 client and implements basic wallet features. You can create a new coin account, execute transactions, check account balance and view transactions./
```

Comment:
The term "api 2.0" by itself is going to confuse people, especially since the API versioning currently starts with 1.x. It's probably best not to mention the version here at all, just call it the "Spacemesh API."

Reply:
done